exporter/exporter.go

```diff
@@ -92,12 +87,15 @@ const (
 
 // Collect sends the collected metrics from each of the collectors to
 // prometheus.
-func (coll windowsCollector) Collect(ch chan<- prometheus.Metric) {
+func (coll WindowsCollector) Collect(ch chan<- prometheus.Metric) {
 	t := time.Now()
-	cs := make([]string, 0, len(coll.collectors))
-	for name := range coll.collectors {
+	collectors := make([]collector.Collector, 0, len(coll.Collectors))
```

Comment:
It doesn't look like this variable is being used anywhere, am i missing something?

Reply:
Refactored out